### PR TITLE
update xml sitemaps

### PR DESCRIPTION
**Pull request by asmega**

### CODE REVIEW

**Comment by Tr4pSt3R** [comment]
:+1:

File: public/category-sitemap.xml [hunk 1]
```diff
@@ -1,299 +1,310 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (munckymagik) [inline public/category-sitemap.xml]: Looks like the `<?xml ...?>` tag has disappeared from all of these docs. Is that deliberate? Otherwise :+1:

**Comment by tomas-stefano** [comment]
:+1:

File: public/category-sitemap.xml (hunk 1 above)
Comment (asmega) [inline public/category-sitemap.xml]: these were the files that were handed over to me. they passed on the w3c validator with a few warnings, so appear to be valid

**Comment by Tr4pSt3R** [comment]
:shipit: :ship:

File: public/category-sitemap.xml (hunk 1 above)
Comment (munckymagik) [inline public/category-sitemap.xml]: Okey dokey.

**Comment by munckymagik** [comment]
:+1:

**Pull request merged**